pkg/config/remote/service/service.go

```diff
@@ -806,6 +821,16 @@ func (s *CoreAgentService) ClientGetConfigs(_ context.Context, request *pbgo.Cli
 	if err != nil {
 		return nil, err
 	}
+
+	expires, err := s.uptane.TimestampExpires()
+	if err != nil {
+		return nil, err
+	}
+	if expires.Before(time.Now()) {
+		log.Errorf("Timestamp expired at %s, flushing cache", expires.Format(time.RFC3339))
```

Comment:
I think we want this to be a warn instead of an error:

1. Customers often have observability set up on the logs of their agents and "Error" tends to raise alarm.
2. Also since this is a situation that we caught and handled, it's more of a warn kind of thing than an actual overall failure.